dipy/denoise/fast_lpca.pyx

```diff
@@ -0,0 +1,286 @@
+import numpy as np
+cimport cython
+cimport numpy as cnp
+
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+@cython.cdivision(True)
+cdef void fast_vecmat_mul(double[:] x, double[:, :] A, double[:] out) nogil:
```

Comment:
Is it possible to pass raw pointers instead of memory views here?:
`fast_vecmat_mul(double *x, double ** A, double* out)`